src/js/components/Grommet/__tests__/__snapshots__/Grommet-test.js.snap

```diff
@@ -22,20 +22,26 @@ exports[`Grommet announce 1`] = `
 
 exports[`Grommet announce 2`] = `
 <div
+  aria-atomic="true"
   aria-live="assertive"
+  data-initial-announce-timeout-id="38"
   id="grommet-announcer"
-  style="left: -100%; right: 100%; position: fixed; z-index: -1;"
+  style="position: absolute; left: -9999px; height: 1px; width: 1px; overflow: hidden;"
```

Comment:
The previous style wasn't 'broken' in that it successfully moved the element off the visible screen. However, it wasn't the most robust or semantically clear way to create a 'screen reader only' element.

clip-path and overflow: hidden creates a truly 'visually hidden' element that remains fully accessible, which is the recommended standard for screen-reader-only content.

https://www.a11yproject.com/posts/how-to-hide-content/